### ISSUE TITLE
#Hacktoberfest

### ISSUE THREAD

**Issue by arguiot** [arguiot/lite.the-scientist.fr#2]
Hacktoberfest

**Comment by arpit1997** [comment]
Is there anything to be done here or it is just a test issue :smile:

**Comment by singh1114** [comment]
I think he was trying to have some fun.. :)

**Comment by arguiot** [comment]
@singh1114 , @arpit1997 I added this issue because of an event that's called Hacktoberfest and you have to put this issue to submit your project.

**Comment by arguiot** [comment]
You can still watch the final result of this project here : [lite.the-scientist.fr](https://www.lite.the-scientist.fr)

**Comment by arpit1997** [comment]
Yeah that is true but in an issue you should mention what is the issue, what is requirements and what needs to be resolved
